src/Asset/Controller/TreeController.php

```diff
@@ -76,7 +76,7 @@ public function __construct(
     #[PageSizeParameter]
     #[ParentIdParameter(
         description: 'Filter assets by parent id.',
-        example: null,
+        example: 83,
```

Comment:
```suggestion
        example: 1,
```
Let's rather use 1 since it is the root folder